src/index.js

```diff
@@ -0,0 +1,289 @@
+  if(radioBtns[0].checked){
+    helperBtnPlus(priceFirstTicketSenior,totalSumOfSeniorModal)
+  } else if(radioBtns[1].checked){
+    helperBtnPlus(priceSecondTicketSenior,totalSumOfSeniorModal)
+  } else if(radioBtns[2].checked){
+    helperBtnPlus(priceThirdTicketSenior,totalSumOfSeniorModal)
+  }
+})
+   
+butM2.addEventListener('click', function(){
+  if(parseInt(countInput2.value) > 1) {
+      btnInputMinus(countInput2, ticketsModalSenior, countInput4,units2)
+      if(radioBtns[0].checked){
+        helperBtnMinus(priceFirstTicketSenior,totalSumOfSeniorModal);
+      } else if(radioBtns[1].checked){
+          helperBtnMinus(priceSecondTicketSenior,totalSumOfSeniorModal);
+      } else if(radioBtns[2].checked){
+           helperBtnMinus(priceThirdTicketSenior,totalSumOfSeniorModal);
+      }
+  }
+})
+
+const openBtn = document.getElementById('openBtn')
+openBtn.addEventListener('click', openModal)
+
+IMask(document.getElementById('phone'),{
+  mask: '+{375}(00)00-00-000',
+})
+
+let wrapper3 = document.querySelector('wr3');
+let countInput3 = document.getElementById('count3');
+let butM3 = document.getElementById('bminus3');
+let butP3 = document.getElementById('bplus3');
+let units3 = countInput3.value.replace(/\d/g, '');
+
+butP3.addEventListener('click', function(){
+  btnInputPlus(countInput3, ticketsModalBasik, countInput, units3);
+  if(selectTicket.value === 'firstT'){
+    helperBtnPlus(priceFirstTicketBasic,totalSumOfBasikModal)
+  } else if(selectTicket.value === 'secondT'){
+    helperBtnPlus(priceSecondTicketBasic,totalSumOfBasikModal)
+  } else if(selectTicket.value === 'thirdT'){
+    helperBtnPlus(priceThirdTicketBasic,totalSumOfBasikModal)
+  }
+})
+
+butM3.addEventListener('click', function(){
+  if(parseInt(countInput3.value) > 1) {
+      btnInputMinus(countInput3, ticketsModalBasik, countInput,units3)
+      if(selectTicket.value === 'firstT'){
+        helperBtnMinus(priceFirstTicketBasic,totalSumOfBasikModal);
+      } else if(selectTicket.value === 'secondT'){
+        helperBtnMinus(priceSecondTicketBasic,totalSumOfBasikModal);
+      } else if(selectTicket.value === 'thirdT'){
+        helperBtnMinus(priceThirdTicketBasic,totalSumOfBasikModal);
+      }
+  }
+})   
+        
+let wrapper4 = document.querySelector('wr4');
+let countInput4 = document.getElementById('count4');
+let butM4 = document.getElementById('bminus4');
+let butP4 = document.getElementById('bplus4');
+let units4 = countInput4.value.replace(/\d/g, '');
+
+butP4.addEventListener('click', function(){
+  btnInputPlus(countInput4, ticketsModalSenior, countInput2, units4);
+    if(selectTicket.value === 'firstT'){
+    helperBtnPlus(priceFirstTicketSenior,totalSumOfSeniorModal)
+  } else if(selectTicket.value === 'secondT'){
+    helperBtnPlus(priceSecondTicketSenior,totalSumOfSeniorModal)
+  } else if(selectTicket.value === 'thirdT'){
+    helperBtnPlus(priceThirdTicketSenior,totalSumOfSeniorModal)
+  }
```

Comment:
Я честно говоря не совсем понимаю зачем у тебя 3 ифа и для всех них используется одно и то же действие.

Ты можешь комбинировать условие в if с помощью логических операторов && либо || (обязательно почитай о них и в чем разница по ссылке ниже)

Почитай про логические операторы в https://learn.javascript.ru/logical-operators

Вот как будет выглядеть код

```suggestion
    if(selectTicket.value === 'firstT' || selectTicket.value === 'secondT' || selectTicket.value === 'thirdT'){
    helperBtnPlus(priceFirstTicketSenior,totalSumOfSeniorModal)
  } 
```

Аналогично исправь для остальных случаев.

Reply:
Извини, только увидел, что меняется аргумент цены. priceFirstTicketSenior, priceSecondTicketSenior и priceThirdTicketSenior. 

У меня есть в мыслях как это сделать лучше. Я напишу как, после того как исправим остальные комменты.

Я посмотрел запустил и посмотрел вопрос насчет респонсив img и насчет includes для тест радио кнопок. Это тоже разберем.